.github/workflows/lint-test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.8]
+        node-version: [14.x, 16.8]
```

Comment:
```suggestion
        node-version: [14.x, 16.x, 18.x]
```

Reply:
The reason we can switch to `16.x` is because the reason for pinning the version no longer applies: https://github.com/MetaMask/eth-sig-util/pull/199